### ISSUE TITLE
can't click on hyprland named workspaces

### ISSUE THREAD

**Issue by jdratlif** [Alexays/Waybar#2883]
When I name my workspaces in hyprland, I see the titles in waybar with the hyprland/workspaces module, but if I click on the name, it does not switch to the workspace. If I have just numeric IDs for the workspace, clicking does change the workspace.

I'm on Fedora 39 with waybar 0.9.24.

Thanks

**Comment by jdratlif** [comment]
[waybar-config.json](https://github.com/Alexays/Waybar/files/14151432/waybar-config.json)

**Comment by jdratlif** [comment]
okay, I must have done something wrong in my config. It's working fine.

**Comment by jdratlif** [comment]
Okay, I found the problem. If I put `|` characters in my workspace name, I cannot click on them. This worked on the sway/workspaces plugin.

**Comment by zjeffer** [comment]
Are there any relevant logs you can share if you turn on trace logging when running Waybar?

Also, can you edit the title to better reflect the issue?

**Comment by jdratlif** [comment]
I edited the title. The workspace name I was using before was 'Slack / Notes / Terminals'. I changed it to 'Slack - Notes - Terminals' and things work fine.

I'm not sure what logs you're referring to. How do I enable trace logging? Do I need to start waybar from the cli with an environment variable or something?

**Comment by zjeffer** [comment]
> I'm not sure what logs you're referring to. How do I enable trace logging? Do I need to start waybar from the cli with an environment variable or something?

Yes, you can start waybar from the cli with the `-l trace` argument, like this:
```
waybar --config <path-to-config.json> --style <path-to-style.css> -l trace
```

**Comment by jdratlif** [comment]
[trace-log.txt](https://github.com/Alexays/Waybar/files/14159453/trace-log.txt)

I updated my hyprland config. I named 4 workspaces.

1: Kitty Terminal - Waybar with Trace Logging
2: Foot / With Slashes
3: Thunar | With Pipes
4: Firefox - With Dashes

I launched kitty, foot, thunar, and firefox on the four workspaces.
I killed waybar (pkill waybar)
I relaunched waybar on workspace "1" with waybar -l trace 2>&1 | tee trace-log.txt
I tried to move between the 4 workspaces. Sometimes with key bindings (MOD + #), sometimes by clicking on waybar.

I can move to any workspace with keybindings.
I can move to 3 of the 4 workspaces with waybar. Workspace "2: Foot / With Slashes" I cannot move to with waybar. I click on the section in waybar and it does nothing.

I see nothing special in the trace logs, but maybe you will see something I don't. It looks like waybar sent the message to hyprland, but it did not work. Other than the slash (/) in the workspace name, I see nothing special about that workspace. I had no problems in Sway, and I have no problems with keybinds in hyprland. This doesn't mean it's not a problem with hyprland, but I thought it made more sense to check with waybar first.

Thanks
